libs/CMakeLists.txt

```diff
@@ -113,6 +113,11 @@ if(TARGET APEX::apex)
   target_link_libraries(hpx_full INTERFACE APEX::apex)
 endif()
 
+# Do I need this here?
```

Comment:
You most likely will not need this as it will be propagated from `hpx_core`.

Reply:
Removed